docs/config/README.md

```diff
@@ -46,9 +46,9 @@ This is the list of prompt-wide configuration options.
 
 ### Options
 
-| Variable       | Default | Description                                                        |
-| -------------- | ------- | ------------------------------------------------------------------ |
-| `add_newline`  | `true`  | Add a new line before the start of the prompt.                     |
+| Variable       | Default                       | Description                                            |
+| -------------- | ----------------------------- | ------------------------------------------------------ |
+| `add_newline`  | `true`                        | Add a new line before the start of the prompt.         |
 | `prompt_order` | [link](#default-prompt-order) | Configure the order in which the prompt module occurs. |
```

Comment:
Recently `starship module --list` flag was added which list all supported modules. So it will be better if docs was updated with this information about where user can get list of all supported values.

Reply:
Sorry, not 100% sure what you mean by that. The docs have the default listing (under default_prompt_order) and `ruby` is already printed by `starship module --list`

Reply:
Currenly user cannot find out what is accepted value for prompt_order through docs. So directing user to `starship module --list` will help user to see all accepted value for prompt_list.

Reply:
Is that specific to the Ruby module, or a general note for the docs?

Reply:
Since "Prompt Order" will (I believe) always have the full list of modules, maybe we don't need to discern between the accepted values and the default list. 🤔

Reply:
Sorry, I'm still a little lost here.

If we're talking about changes to the overall documentation structure or code structure, I'm all for it (I *really* don't like having three copies of the same information floating around in the codebase), but that seems like a separate issue.

Reply:
We could continue this discussion elsewhere. 😛
I'll merge this for now.